benchmarks/benchmarks/sparse_csgraph_djisktra.py

```diff
@@ -0,0 +1,41 @@
+"""benchmarks for the scipy.sparse.csgraph module"""
+from __future__ import print_function, absolute_import
+import numpy as np
+import scipy.sparse
+
+try:
+    from scipy.sparse.csgraph import dijkstra
+except ImportError:
+    pass
+
+from .common import Benchmark
+
+
+class Dijkstra(Benchmark):
+    params = [
+        [30, 300, 900]
+    ]
+    param_names = ['n', 'format', 'normed']
+
+    def setup(self, n):
+        # make a random connectivity matrix
```

Comment:
Quite a few of our benchmarking modules do manually set the random seed. Are we confident that the `__init__` file is actually always going to do that? What about the case where `asv` is used directly instead of our in-house `run.py`?

Reply:
added a random seed setting